static-assets.md

```diff
@@ -0,0 +1,19 @@
+## Static assets
```

Comment:
Can you update to follow the same format/template that's used in the other sections.